### ISSUE TITLE
Clementine deletes automatically deleted tag information

### ISSUE THREAD

**Issue by InterLinked1** [clementine-player/Clementine#6428]
### Before posting

Please follow the steps below and check the boxes with [x] once you did the step.

- [X] I checked the issue tracker for similar issues
- [ ] I checked the [changelog](https://github.com/clementine-player/Clementine/blob/master/Changelog) if the issue is already resolved
- [ ] I tried the latest Clementine build from [here](https://builds.clementine-player.org/)

### System information

Please provide information about your system and the version of Clementine used.

- Operating System: Windows 10 Professional 17736.805
- Clementine version: 1.3.1

### Expected behaviour / actual behaviour

I went through and added metadata for songs I added to playlists, specifically to have the "Year" available. When songs are played, and the selection is changed, Clementine deletes the metadata that was previously added, *irreversibly*. The only way to get it back is to start over again by re-adding the metadata using "Complete Tags Automatically".

This doesn't happen if you stop a track and then move on to a different one, but if you just let a playlist run or double-click a different track without stopping the first one, it will wipe out all of the metadata.

### Steps to reproduce the problem (only for bugs)

Import some songs and use the "Complete Tags Automatically" feature to add metadata for them and let the playlist run: Clementine will delete all the metadata as it plays each song.

**Comment by JulianVolodia** [comment]
Hi @InterLinked1 , are you developer? Could you try to prepare patch?

**Comment by InterLinked1** [comment]
> Hi @InterLinked1 , are you developer? Could you try to prepare patch?

No, unfortunately, I'm not, at least I'm not a software developer. I just downloaded the software today and decided to organize my music library (up until now, I've just used Windows Media Player). I was really looking for something that looked "Windows"ish that could reliably store metadata about the year. This is close, but there is definitely some bug here.)

Anyways, I have no idea what the exact issue is or how to fix it, but wanted to raise a flag about it.

**Comment by JulianVolodia** [comment]
Please, use Clementine on some copy if you could. We try the best to make it better and fix everything for you.

If you see any issue, tell us.

Please, tell me if you could use debug version and provide full logs for us. If you don't know if they contain any private data, try to copy only few lines of the end of it, and check this lines manually.
https://builds.clementine-player.org/win32/debug/

Best and thank You for your effort to help us.

**Comment by JulianVolodia** [comment]
P.S. of course if you have some time and want to have new hobby try to look into source and try to see how it works in background. Maybe you could even help with do it. Really, this is not so hard, this needs persistance to learn code (a lot of, but still you could always help us).

**Comment by InterLinked1** [comment]
I can program, but I'm certainly not very good at C. I do more scripting and web stuff, honestly, mostly PHP and special-purpose programming languages these days. Regardless, I don't have the time to help with this, unfortunately.

I can't really install a debug version. It's easy enough for anyone to reproduce, though, if that helps.

Also worth mentioning is the Complete Tags Automatically feature doesn't work for more than 4 or 5 songs at a time. More than that, and the rest will fail. The result was for all of my 336 songs, I had to go through and select them in groups of 4 to 5, and do Complete Tags Automatically, confirm the information was correct, OK. And sometimes 1 or 2 of them would fail. I would think this could be designed so it could do this kind of thing automatically.
